lib/verify.js

```diff
@@ -1,82 +1,101 @@
 'use strict';
 
+const fileType = require('file-type');
 const Validator = require('validator');
-const FileType = require('file-type');
-const {logger} = require('./logging');
+
+const { logger } = require('./logging');
 
 const MAX_TITLE_LENGTH = 140;
 
 class VerificationError extends Error {
-    constructor (message, metadata=null) {
+    constructor(message, metadata = null) {
         super(message);
+
         this.name = this.constructor.name;
+
         Error.captureStackTrace(this, this.constructor);
+
         if (metadata) {
             this.metadata = metadata;
         }
     }
 }
 
-exports.VerificationError = VerificationError;
-
-function assert(value, message, metadata=null) {
-    if (value) {
+function assert(value, message, metadata = null) {
+    if (value === true) {
```

Comment:
Do we have a lint rule that requires explicit check for true/false instead of using truthy/falsy ? Or is it needed for this validation only?

Reply:
This check made no sense to be truthy since we always expect booleans to be passed in so I put in a `true` equality. It wasn't a linting issue.